superset-frontend/package.json

```diff
@@ -12,12 +12,12 @@
     "test": "NODE_ENV=test jest",
     "type": "tsc --noEmit",
     "cover": "NODE_ENV=test jest --coverage",
-    "dev": "webpack --mode=development --colors --progress --debug --watch",
-    "dev-server": "NODE_ENV=development BABEL_ENV=development node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --mode=development --progress",
```

Comment:
I don't mind it on `dev` and `dev-server` as it's something to watch, and helps me gauge when I should see the view refresh.

Reply:
They will continue to report progress, as long as the environment variable `CI` is not set——which should be the case for most If not all users. This change is just for consistency’s sake.